### PR TITLE
Docsp 7601 insert many

### PR DESCRIPTION
**Pull request by mongomoe**
Ticket: https://jira.mongodb.org/browse/DOCSP-7601
Stage: https://docs-mongodbcom-staging.corp.mongodb.com/node/docsworker/DOCSP-7601-insertMany/usage-examples/insertMany

### CODE REVIEW

File: source/usage-examples/insertMany.txt [hunk 1]
```diff
@@ -0,0 +1,31 @@
+=====================
+Create Many Documents
+=====================
+
+.. default-domain:: mongodb
+
+You can create multiple documents using the `insertMany()
+<https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#insertMany>`_
+method. The ``insertMany()`` method inserts multiple documents in the specified collection 
+by taking in an array of documents.
```
Comment (cbush) [inline source/usage-examples/insertMany.txt]: Minor: I find this phrasing a bit odd. Maybe something like:

>The ``insertMany()`` method *takes an array of documents* to insert into the specified collection.

File: source/usage-examples/insertMany.txt [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+<https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#insertMany>`_
+method. The ``insertMany()`` method inserts multiple documents in the specified collection 
+by taking in an array of documents. 
+
+Create an `Object
+<https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object>`_
+to specify additional options.  Set the ``bypassDocumentValidation`` field to bypass
+:manual:`schema validation <core/schema-validation/>` rules in MongoDB 3.2 or higher. 
+Specify ``ordered:true`` to prevent inserting the remaining documents if a 
+previous document in the array fails to be inserted.
```
Comment (cbush) [inline source/usage-examples/insertMany.txt]: Active voice: 

>if the insertion failed for a previous document in the array.

File: source/usage-examples/insertMany.txt (hunk 1 above)
Comment (mongomoe) [reply source/usage-examples/insertMany.txt]: That does sound better - fixed verbatim

File: source/usage-examples/insertMany.txt (hunk 2 above)
Comment (mongomoe) [reply source/usage-examples/insertMany.txt]: Good catched - fixed

**Pull request merged**